semeio/jobs/scripts/auto_correlate_observations.py

```diff
@@ -0,0 +1,51 @@
+import collections
+import yaml
+
+from ert_data.measured import MeasuredData
+from ert_shared.libres_facade import LibresFacade
+from res.enkf import ErtScript
+
+from semeio.jobs import auto_correlate_observations
+from semeio.jobs.scripts.correlated_observations_scaling import (
+    CorrelatedObservationsScalingJob,
+)
+
+
+class AutoCorrelateObservationsJob(ErtScript):
+    def run(self, *args):
+        config_record = _fetch_config_record(args)
```

Comment:
Why we need an extra function `_fetch_config_record` as this only loads the file?

Reply:
Because I'm designing for replacement and abstraction, not for reuse :) And since I'm later not expecting `args` to point to files, but to contain a stream of data, I've separated that part of the code from the core logic 🤷